.readthedocs.yml

```diff
@@ -2,11 +2,6 @@
 # See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 version: 2
 
-build:
```

Comment:
Are you sure about the RTD one?

Reply:
https://github.com/pymc-labs/pymc-marketing/blob/1da568216ad071bc4b7ee034c551e12f73f68d18/.readthedocs.yml#L5-L16

I was getting the following warning that it's duplicated:

<img width="636" alt="image" src="https://user-images.githubusercontent.com/24764859/215230247-633df74d-66dd-4386-a3e7-15e550755ce2.png">

Reply:
> Are you sure about the RTD one?

In other words, not really...

Reply:
CC @OriolAbril

Reply:
Yes, that should only be there one time